Description: Do not include the native libraries in the main jar
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -474,6 +474,7 @@
               <!-- Note that no terminal "*" is included in this list,
                  which will force failure on unsupported platforms.
               -->
+<!--
               <attribute name="Bundle-NativeCode"
                          value="
 com/sun/jna/win32-x86/jnidispatch.dll;
@@ -551,6 +552,7 @@
 com/sun/jna/darwin-aarch64/libjnidispatch.jnilib;
 osname=macosx;processor=aarch64
 "/>
+-->
           </manifest>
           <manifest file="@{target}" mode="update" if:true="@{module-info}">
               <attribute name="Multi-Release" value="true"/>
@@ -581,6 +583,7 @@
       <fileset dir="${classes}" excludes="${jar.omitted}">
         <patternset refid="jar-compiled"/>
       </fileset>
+<!--
       <zipfileset src="${lib.native}/win32-x86.jar"
                   includes="*jnidispatch*"
                   prefix="com/sun/jna/win32-x86"/>
@@ -677,6 +680,7 @@
       <zipfileset src="${lib.native}/w32ce-arm.jar"
                   includes="*jnidispatch*"
                   prefix="com/sun/jna/w32ce-arm"/>
+-->
       <zipfileset dir="."
                   includes="LICENSE,LGPL2.1,AL2.0"
                   prefix="META-INF"/>
